web/web.go

```diff
@@ -0,0 +1,2 @@
+// This package collects types that are common to both wfe and wfe2.
+package web
```

Comment:
What's this empty file for?

Reply:
I added this to have a place to document what this package is for. I could move the comment to one of the nonempty files; what do you think?

Reply:
The general way I've seen this done is to have a specific `docs.go` file but I don't think that's a concrete convention.